src/main/java/pl/oskarkowalski/recruitment/ticketbookingapp/entities/Person.java

```diff
@@ -0,0 +1,41 @@
+package pl.oskarkowalski.recruitment.ticketbookingapp.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Entity
+@Table(name="PERSONS")
```

Comment:
to jest persons czy raczej people, a może viewers? dlaczego tak nazwa? w jaki sposób jest związana z domeną?

Reply:
W ogóle - po dokonaniu commitu zdałem sobie sprawę, że w ogóle encja Persons/People/Viewers jest do wycięcia - na potrzeby aplikacji i zadania wystarczy w zupełności imię i nazwisko (przyszłościowo również prawdopodobnie adres e-mail - w celu realizacji zadania dodatkowego) w rezerwacji.